README.md

```diff
@@ -12,4 +24,4 @@ a live stream of the discussion and a recording of past meetings. Topics for the
 meetings are determined on an as-needed basis and may be technical or management
 of nature.
 
-Current meeting schedule: Wednesdays 23:30 UTC
+Normal meeting schedule: Wednesdays 23:30 UTC
```

Comment:
Rather than change this and add need to add the further notice warnings, etc. What if (especially if they are going to end up as "ac-hoc" meetings) we just don't call the Thursdays meetings Express.js TC Meetings? Maybe like Express.js Roadmap Meetings? This is not a request to change anything if you don't want here, just a thought I had regarding it. I do kind of feel bad that, if this is the TC meeting, having it moved where a key member (myself) cannot even commit to being there due to conflicting with work. I just cannot jeopardize my job that puts my roof over my head for something that does not, if that makes sense, haha

Reply:
@dougwilson  - on the other hand, express 5 being important to the ecosystem, the community and everyone involved here, it is critical that express TC is committed to its execution plan from start to end, driving from the front. For that reason, I would maintain that we call it and run it as Express TC meetings; not ad-hoc, and make any adjustments possible to accommodate key persons like you.

Reply:
I agree it is of up-most important and 100% want to be at the meetings... but I'm not sure if the meeting time/day is up for reconsideration currently as I have brought up the conflict before and the meeting remains at a conflicting time, haha

Reply:
My thought was if this is the "TC Meeting" then should the schedule be set to try and accommodate all the TC members at most and others can attend if able who are not in the TC? If the meeting is set to accommodate more than the TC members alone, that feels like it is no longer a TC meeting, but a more general, wider, Express.js meeting (which I have nothing against, of course).

Reply:
@dougwilson - no other TC members than you, have indicated any inconvenience to the current time, IIUC.

The meeting is never set to accommodate non-TC members alone, I don't know how you got that impression?

Reply:
AFAIK the existing TC meeting didn't have any issues with the current TC members. What was the reason for changing the meeting time/day in the first place? Why not go back to the time in the README?

Reply:
> The meeting is never set to accommodate non-TC members alone, I don't know how you got that impression?

My impression came from #95 , unless I am completely misunderstanding that issue?

Reply:
> What was the reason for changing the meeting time/day in the first place? Why not go back to the time in the README?

@dougwilson - answer in your own comment
https://github.com/expressjs/discussions/issues/99#issuecomment-591204805

Reply:
Yea, I am open to it changing. But why was it changed to when I can't actually attend, lol

Reply:
> My impression came from #95 , unless I am completely misunderstanding that issue?

right - but that issue does not propose to accommodate non-TC at the cost of keeping TC away. May be I misunderstood your original comment as hinting on these lines, but reading it again, I don't think it was put that way.

Let us all together uphold the value of express TC, the weight of discussions happened there, and be accountable for decisions taken in the meetings - for the interest of the community and the users that depend on it.

Reply:
Yep, I agree. Does this mean we can move the TC meeting back to the original time, or am I just never going to be able to attend a TC meeting for the foreseeable future :) ?

Reply:
> Let us all together uphold the value of express TC, the weight of discussions happened there, and be accountable for decisions taken in the meetings - for the interest of the community and the users that depend on it.

Also, I'm not sure what exactly that means, as it almost sounds like if a decision is reached in a meeting is somehow has more weight than otherwise, which is not what our currently process is. The process is outlined inhttps://github.com/expressjs/express/blob/master/Contributing.md#tc-process which states "The group tries to find a resolution that has no open objections among TC members."; a TC member can have an objection to something even if they are not present in a TC meeting, as otherwise it would not make sense to hold one without all members present.

There is a process for when a TC member as an open objection and a consensus cannot be reached, which is through the majority vote method (has never been used before). There are currently 5 active TC members so only 3 votes would be necessary for a majority, if we actually have to do such a thing.

Reply:
are you talking about the timing decisions? I don't think we made it as a `TC decision`, but a consensus among members present.

So how about a doodle method? if that does not work out well, let us consider invoking the voting method as well.

Reply:
I'm referring to your specific statement -- "accountable for decisions taken in the meetings"; that goes against the current governance, as a decision cannot be final inside a meeting unless there is actually consensus among all the TC members. It also means that if not all members are present for a meeting, by nature a consensus cannot be reached during a meeting...

Reply:
ok, that statement cannot be cut in the middle. It is prepended with `value of express TC` and appended with `for the interest of the community` , and has to be read together, as is.

> a TC member can have an objection to something even if they are not present in a TC meeting, as otherwise it would not make sense to hold one without all members present.

> a decision cannot be final inside a meeting unless there is actually consensus among all the TC members. It also means that if not all members are present for a meeting, by nature a consensus cannot be reached during a meeting...

these are not present in the clauses, and are just extrapolations. Strictly speaking, incorrect.

Reply:
Yes, they are an extrapolation, but are, strictly speaking, 100% correct. You see, if something is decided in a meeting without all the TC members present, then those members do not have the opportunity to object to it at that time. So, they are 100% allowed to object after the meeting took place and they have listened to it / read the notes on it. Once there is an objection, there is not consensus. But during the meeting without all members present, it is _unknown_ if there is consensus at that point.

Reply:
> You see, if something is decided in a meeting without all the TC members present, then those members do not have the opportunity to object to it at that time.

I understand the sentiment, but the clauses do not mention anything about absentees and their rights; which makes sense too. Otherwise no meaningful discussion and consensus is possible in a meeting if a member is missing.

> Once there is an objection, there is not consensus

makes sense, but `objection` should be better qualified and scopped -  within the meeting, while consensus seeking process is on. Decisions cannot be postponed indefinitely, in expectation of future objections?

Reply:
I'm not saying that meetings cannot try and make decisions. Only that just because they tried does not mean that a TC member who was or was not there (one who was there can change their mind of course) can later object. And you cannot dismiss their objection simply because they were unable to attend the meeting, especially if they expressed being unable to before hand and there wasn't an opportunity to take part in the discussion before-hand in an async fashion like over Github.

Just as you state that the clauses do not mention anything about absentees and their rights, it too does not state anywhere that decisions made in a meeting carry any more weight than done anywhere else, nor does it say that they are final in a meeting, etc.

All discussions are meaningful, no matter what medium they occur in.

> makes sense, but objection should be better qualified and scopped - within the meeting, while consensus seeking process is on. Decisions cannot be postponed indefinitely, in expectation of future objections?

I'm not sure what this mean. Again, you seem to be arguing that some specific things are not explicitly in the clause, and this, too, is not specifically in the cause. There is no definition for when a consensus seeking process starts or stops. The closest you have is that it calls out specifically about objections existing. 

> Decisions cannot be postponed indefinitely, in expectation of future objections?

I have never said that, either. Once something is spoken of, it is reasonable to proceed as if there is a presumed consensus. If an object does happen to appear in the future, and whatever the thing is hasn't actually be done, then it's clearly no longer a consensus on something that hasn't even happened yet.

If the ideal state you would like to see is to reduce the chance of objections post-meeting, it seems there are some pretty easy paths there (this is not comprehensive): you can attempt to schedule the meeting when those likely to object can attend, or you can open a discussion over some async meeting like Github  prior to the meeting so objections can be worked through. Working through objects can occur over any medium, including meetings themselves.

One of the main purposes of recording meetings is in order to allow the TC members to reach consensus by giving the opportunity to those who were unable to take part to understand what is happening and raise objections. This is similar to the requirement for pull requests to wait for a period of time, even when reaching the needed number of approvals, such that one has an opportunity to raise an objection, if they wish.

I would never argue that any decision can just be objected to with indefinite time, as that is arguing a nonsensical extreme. But if a TC member raises an objection within the same time period as a PR has to wait (36 hours) after the meeting's recording is uploaded for listening, I think that is a very reasonable amount of time, do you disagree?

Reply:
@dougwilson - +1 to almost everything you said, except:

> you can attempt to schedule the meeting when those likely to object can attend

may be this is impractical to achieve? how do one know in advance that someone is likely to object?

And moreover, since we are riding on a path that require high level of engagement and commitment leading up to a series of decision making, the natural expectation is that people are positive, progressive and accommodative by default.

Let us move on! I have seen you set up a TC at the regular schedule. It is 5 AM for me, but will surely accommodate - really excited to take part in moving this forward!

Reply:
> may be this is impractical to achieve? how do one know in advance that someone is likely to object?

Correct, it is not really possible to _know_ who will object. That is why I specified "those who are likely to object" -- perhaps one way think that based on prior conversations / etc. I think it would be unreasonable to actually be able to know that, of course. Just as it is reasonable to say you don't know who would object, it is reasonable to give some small window for objection post-meeting is all.

> the natural expectation is that people are positive, progressive and accommodative by default.

I 100% agree, and unless there is something else going on here (perhaps through tone projection onto text, etc.)? I'm not aware of anyone acting in a way trying to _prevent_ progressing movement. Someone disagreeing on a direction, especially with an alternative direction presented, is by nature acting in good faith. Throwing up an objection without providing a reason would be obstructing; if the person who objects does provide a reason, but it isn't clear, it is fair to ask them to better clarify on it and provide specific questions to help drive the clarity for both sides to better understand each other -- they may actually be in agreement in the end, but saying the same thing in two different ways.

Reply:
It sounds to me like we need to revisit the scheduling of the v5 meetings. 

From what I recall, we put forward meeting at the time outlined here after the first ad hoc meeting #103 based on the preferences expressed by those in attendance. If that time does not work for TC members and main stakeholders, then it makes sense to try and get consensus on another time. 

We want/need @dougwilson to be able to attend in order for meaningful progress to be possible. I hope we can also accommodate colleagues in timezones in the eastern hemisphere, and @gireeshpunathil your flexibility is greatly appreciated. 

If someone can start a doodle or equivalent to try and find a suitable meeting time to settle on, that would be awesome. I'm on the East coast of the US and very flexible personally, so I'll defer to the preferences of those we wish to accommodate. I don't want this to be a blocker for progress, and again thank you @gireeshpunathil for being so flexible on this issue currently, but I would be happy if we find a time that is palatable to all parties.